docker/docker-compose.yml

```diff
@@ -82,12 +82,29 @@ services:
       container_name: lnd_ltc
       links:
           - "ltcd:blockchain"
+      volumes:
+          - lnd_ltc_data:/root/.lnd
 
     lnd_btc:
       extends: lnd
       container_name: lnd_btc
       links:
           - "btcd:blockchain"
+      volumes:
+          - lnd_btc_data:/root/.lnd
```

Comment:
Same comment here.  shouldn't require it this volume if you create it in `lnd/Dockerfile`